pandas/tests/extension/base/missing.py

```diff
@@ -19,7 +19,7 @@ def test_isna(self, data_missing):
 
         # GH 21189
         result = pd.Series(data_missing).drop([0, 1]).isna()
-        expected = pd.Series([], dtype=bool)
+        expected = pd.Series([], dtype=bool, index=pd.RangeIndex(0))
```

Comment:
Why is this a RangeIndex? I would think it's a regular index?

Reply:
The index type of `result` stayed the same, it's `RangeIndex`on master and it is in this PR. But since the default index type of `pd.Series([])` is adjusted here, expected suddenly would have a different index type. Therefore I need to pass it a `RangeIndex` explicitly.